packages/components/src/font-size-picker/stories/index.js

```diff
@@ -0,0 +1,65 @@
+/**
+ * WordPress dependencies
+ */
+import { useState } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import FontSizePicker from '../';
+
+export default { title: 'FontSizePicker', component: FontSizePicker };
+
+const FontSizePickerWithState = ( { ...props } ) => {
```

Comment:
Nit, if it doesn't use any particular props, it can be simplified to:
```suggestion
const FontSizePickerWithState = ( props ) => {
```

Reply:
Updated